recipes/sphinx-pypi-upload/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{%set name = "Sphinx-PyPI-upload" %}
+{%set version = "0.2.1" %}
+{%set compress_type = "tar.gz" %}
+{%set hash_type = "sha256" %}
+{%set hash_val = "5f919a47ce7a7e6028dba809de81ae1297ac192347cf6fc54efca919d4865159" %}
+{%set build_num = "0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.{{ compress_type }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ compress_type }}
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  skip: True  # [py3k]
+  preserve_egg_dir: True
+  entry_points:
+    - upload_sphinx = sphinx_pypi_upload:UploadDoc
+
+  number: {{ build_num }}
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - setuptools
+
+test:
+  imports:
+    - sphinx_pypi_upload
+
+  commands:
+    # - upload_sphinx
+    # Can't test without inputs
```

Comment:
Is there isn't a `-h` or some sort of help to test if the script exists?  If not just remove the `commands` test field.